src/NuGet.Core/NuGet.Build.Tasks/GetRestoreSolutionProjectsTask.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
```

Comment:
We should remove it from all files to reduce the daily overhead of this or automate it. Currently the tool from .NET that can do this doesn't work on PackageReference or project.json projects.